schema/index.js

```diff
@@ -11,7 +14,26 @@ const resolvers = {};
   });
 });
 
-let rootSchema = '';
+resolvers.ObjID = new GraphQLScalarType({
+  name: 'ObjID',
+  description: 'Id representation, based on Mongo Object Ids',
+  parseValue(value) {
+    return ObjectId(value); // eslint-disable-line new-cap
```

Comment:
Oh. That was the other one (`no-underscore-dangle`). Should I disable this rule too? 

It would be best if our generated output passes linting. So actually I *would* want it in the generated output, or else to disable the rule.